android/app/src/main/java/com/expensify/chat/RNTextInputResetModule.java

```diff
@@ -0,0 +1,46 @@
+package com.expensify.chat;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.Callback;
+import com.facebook.react.uimanager.UIManagerModule;
+import com.facebook.react.uimanager.UIBlock;
+import com.facebook.react.uimanager.NativeViewHierarchyManager;
+import android.content.Context;
+import android.view.View;
+import android.widget.TextView;
+import android.view.inputmethod.InputMethodManager;
+import android.util.Log;
+
+public class RNTextInputResetModule extends ReactContextBaseJavaModule {
+
+    private final ReactApplicationContext reactContext;
+
+    public RNTextInputResetModule(ReactApplicationContext reactContext) {
+        super(reactContext);
+        this.reactContext = reactContext;
+    }
+
+    @Override
+    public String getName() {
+        return "RNTextInputReset";
+    }
+
+    // Props to https://github.com/MattFoley for this temporary hack
+    // https://github.com/facebook/react-native/pull/12462#issuecomment-298812731
+    @ReactMethod
+    public void resetKeyboardInput(final int reactTagToReset) {
+        UIManagerModule uiManager = getReactApplicationContext().getNativeModule(UIManagerModule.class);
+        uiManager.addUIBlock(new UIBlock() {
+            @Override
+            public void execute(NativeViewHierarchyManager nativeViewHierarchyManager) {
+                InputMethodManager imm = (InputMethodManager) getReactApplicationContext().getBaseContext().getSystemService(Context.INPUT_METHOD_SERVICE);
+                if (imm != null) {
+                    View viewToReset = nativeViewHierarchyManager.resolveView(reactTagToReset);
+                    imm.restartInput(viewToReset);
+                }
+            }
+        });
+    }
+}
```

Comment:
This needs EOL.

Reply:
EOL??

Reply:
Sorry for the confusion, Github marks it as missing when reviewing!

Reply:
So should I do anything further here?

Reply:
You can add empty line at the end of the line.